policybot/dashboard/topics/postsubmit/topic.go

```diff
@@ -32,15 +32,18 @@ import (
 	"github.com/robfig/cron/v3"
 )
 
-// PostSubmit lets users visualize critical information about the project's outstanding pull requests.
+// PostSubmit lets users visualize critical information about which features have been covered and which not after Github PR merged.
 type PostSubmit struct {
 	store         storage.Store
 	cache         *cache.Cache
 	router        *mux.Router
 	latestbaseSha *template.Template
 	baseSha       *template.Template
 	analysis      *template.Template
-	choosesha     string
+	choosesha     string //basesha user choose to view
+	chooseEnv     string //environment of selected basesha user wants to generate detailed test name
+	chooseLabel   string //label of selected basesha user wants to generate detailed test name
+
 }
 
 type LatestBaseShaSummary struct {
```

Comment:
Can we throw comments on public things (in go this is everything that starts with a capital letter) that explain what they do?